arithmetization/src/main/java/net/consensys/linea/zktracer/module/shakiradata/ShakiraData.java

```diff
@@ -40,6 +41,7 @@ public class ShakiraData implements Module {
   private final ShakiraDataComparator comparator = new ShakiraDataComparator();
 
   private final Sha256Blocks sha256Blocks;
+  private final Keccak keccak;
```

Comment:
That was where keccak was ignored previously ?

Reply:
it was a TODO for one day, yes